server/app/middlewares/auth.js

```diff
@@ -0,0 +1,24 @@
+import validator from 'email-validator';
+
+
+const authNewUser = (req, res, next) => {
+  const username = req.body.username.trim();
+
+  if (!req.body.email || !req.body.password || !req.body.username || !req.body.address) {
```

Comment:
The exact field that is missing from the input might be more helpful here for the user. What do you think?